pages/docs/reference/sealed-classes.md

```diff
@@ -48,3 +96,19 @@ fun eval(expr: Expr): Double = when(expr) {
 }
 ```
 </div>
+
+## Experimental status of sealed interfaces and extended inheritance rules
+
+[Sealed interfaces](#sealed-interfaces) and [extended inheritance rules](#extended-inheritance-rules) are [Experimental](evolution/components-stability.html).
+To be able to use them in your code, switch to the language version `1.5`:
+* In Gradle, add the [compiler option](using-gradle.html#attributes-common-for-jvm-and-js) `languageVersion` with the value `1.5`.
+
+<div class="sample" markdown="1" mode="groovy" theme="idea">
+
+```groovy
```

Comment:
Does it work for Kotlin in the same way as in Groovy?

Reply:
Yes, this line is the same in both languages.